### ISSUE TITLE
Dot produces no Graphs

### ISSUE THREAD

**Issue by kokolorix** [doxygen/doxygen#6990]
Errors in log:

```
error: Problems running dot: exit code=1, command='C:\Program Files (x86)\Graphviz2.38\bin\dot', arguments='"S:/Dev/Work/ProdXE7_Help/BrKompHelp/html/de/d8f/class_t_am_leistungen_aa8a735428455f00492fec60c1760d54c_aa8a735428455f00492fec60c1760d54c_cgraph.dot" -Tsvg -o "S:/Dev/Work/ProdXE7_Help/BrKompHelp/html/de/d8f/class_t_am_leistungen_aa8a735428455f00492fec60c1760d54c_aa8a735428455f00492fec60c1760d54c_cgraph.svg"'
error: Problems running dot: exit code=1, command='C:\Program Files (x86)\Graphviz2.38\bin\dot', arguments='"S:/Dev/Work/ProdXE7_Help/BrKompHelp/html/de/d8f/class_t_am_leistungen_a051d755ab11582bed4ac429fc9c2f5da_a051d755ab11582bed4ac429fc9c2f5da_cgraph.dot" -Tsvg -o "S:/Dev/Work/ProdXE7_Help/BrKompHelp/html/de/d8f/class_t_am_leistungen_a051d755ab11582bed4ac429fc9c2f5da_a051d755ab11582bed4ac429fc9c2f5da_cgraph.svg"'
error: Problems running dot: exit code=1, command='C:\Program Files (x86)\Graphviz2.38\bin\dot', arguments='"S:/Dev/Work/ProdXE7_Help/BrKompHelp/html/db/ddc/class_t_kalender_params_a913bd20066c0a0530343be312f4b3e35_a913bd20066c0a0530343be312f4b3e35_icgraph.dot" -Tsvg -o "S:/Dev/Work/ProdXE7_Help/BrKompHelp/html/db/ddc/class_t_kalender_params_a913bd20066c0a0530343be312f4b3e35_a913bd20066c0a0530343be312f4b3e35_icgraph.svg"'
error: Failed to rename file S:/Dev/Work/ProdXE7_Help/BrKompHelp/html/de/d8f/class_t_am_leistungen_aa8a735428455f00492fec60c1760d54c_aa8a735428455f00492fec60c1760d54c_cgraph.svg to S:/Dev/Work/ProdXE7_Help/BrKompHelp/html/de/d8f/class_t_am_leistungen_aa8a735428455f00492fec60c1760d54c_aa8a735428455f00492fec60c1760d54c_cgraph.svg.tmp!
```

From command line:
```
S:\Dev\Work\ProdXE7\Help>dot "S:/Dev/Work/ProdXE7_Help/BrKompHelp/html/de/d8f/class_t_am_leistungen_aa8a735428455f00492fec60c1760d54c_aa8a735428455f00492fec60c1760d54c_cgraph.dot" -Tsvg -o "S:/Dev/Work/ProdXE7_Help/BrKompHelp/html/de/d8f/class_t_am_leistungen_aa8a735428455f00492fec60c1760d54c_aa8a735428455f00492fec60c1760d54c_cgraph.svg"
Error: S:/Dev/Work/ProdXE7_Help/BrKompHelp/html/de/d8f/class_t_am_leistungen_aa8a735428455f00492fec60c1760d54c_aa8a735428455f00492fec60c1760d54c_cgraph.dot: syntax error in line 115 near '.'

Line 115:
  Node57 [label="TKalenderParams::Umwandle\lHTMLFuerKalenderEintrag",height=0.2,width=0.4,color="black", fillcolor="white", style="filled",URL="$db/ddc/class_t_kalender_params_a913bd20066c0a0530343be312f4b3e35.html#a913bd20066c0a0530343be312f4b3e35",tooltip="Neue Zeilen aus GUI mit nichts ersetzen und \"\\" verdoppeln."];
```

```
The Comment:
//---------------------------------------------------------------------------
//!Neue Zeilen aus GUI mit nichts ersetzen und "\" verdoppeln
// ----------------------------------------------------------
```

**Comment by albert-github** [comment]
- Can you please attach a, small, self contained example (source+configuration file in a tar or zip) that allows us to reproduce the problem?  Please don't add external links as they might not be persistent.
- Please also specify the doxygen version used.

**Comment by kokolorix** [comment]
[DoxygenBug.zip](https://github.com/doxygen/doxygen/files/3182333/DoxygenBug.zip)

**Comment by albert-github** [comment]
Problem is the `"\"`  currently the only thing that works, as a workaround, is `` `\` `` or `'\'`, so use a backtick or single quote instead of a double quote.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #6991

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of a release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.16.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
